get-firo/download/index.html

```diff
@@ -66,9 +66,8 @@ <h3>{% t download.qt %}</h3>
                                     </div>
                               </div>
                             </div>
-                            <div class="source">
-                                <a href="https://github.com/firoorg/firo" target="_blank"><i class="icon-github"></i></a>
-                                <a href="https://sourceforge.net/projects/firoorg/" target="_blank"><i class="icon-sourceforge"></i></a>

```

Comment:
We need to keep Sourceforge since it is still being used, but as it is not visible on the site right now it is okay to remove

Reply:
I can comment it out if preferred

Reply:
We can remove